config/environments/production.rb

```diff
@@ -62,8 +62,21 @@
 
   # Ignore bad email addresses and do not raise email delivery errors.
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
-  # config.action_mailer.raise_delivery_errors = false
-
+  Rails.application.configure do
+    config.action_mailer.delivery_method = :smtp
+    config.action_mailer.raise_delivery_errors = true
+    config.action_mailer.perform_deliveries = true
+    config.action_mailer.smtp_settings = {
+        address: ENV["MAIL_ADDRESS"],
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.